specifyweb/frontend/js_src/lib/hooks/useResourceValue.tsx

```diff
@@ -132,7 +132,11 @@ export function useResourceValue<
       );
       if (field === undefined) return;
 
-      if (!parseResults.isValid) setBlockers([parseResults.reason]);
+      if (parseResults.isValid) {
+        setBlockers([]);
+      } else {
+        setBlockers([parseResults.reason]);
+      }
```

Comment:
The way it was in https://github.com/specify/specify7/commit/6bdeba0106e69cbd41c2018cfaec1a7a839e2297#diff-0d189e32be0ec55985090bda95c99a9ae7d87706461a2b9667242015a904f916R134-R135:

```suggestion
      // This assumes that there are no field blockers set by anything else
      if (parseResults.isValid) setBlockers([]);
      else setBlockers([parseResults.reason]);
```

The comment about "This assumes that there are no field blockers set by anything else" is also a good point.
Do not know if it causes issues anywhere yet, but it might at some point.
Before the rewrite to TypeScript in 2021, each save blocker had a unique key, thus making such overwriting less likely. I removed the keys during TypeScript rewrite to make things simpler, but we might want to add them back at some point